packages/ui/src/assets/normalize.scss

```diff
@@ -0,0 +1 @@
+a,abbr,acronym,address,applet,article,aside,audio,b,big,blockquote,canvas,caption,center,cite,code,dd,del,details,dfn,div,dl,dt,em,embed,fieldset,figcaption,figure,footer,form,h1,h2,h3,h4,h5,h6,header,hgroup,i,iframe,img,ins,kbd,label,legend,li,mark,menu,nav,object,ol,output,p,pre,q,ruby,s,samp,section,small,span,strike,strong,sub,summary,sup,table,tbody,td,tfoot,th,thead,time,tr,tt,u,ul,var,video{margin:0;padding:0;border:0;font-size:100%;font:inherit;vertical-align:baseline}article,aside,details,figcaption,figure,footer,header,hgroup,menu,nav,section{display:block}body{line-height:1}ol,ul{list-style:none}blockquote,q{quotes:none}blockquote:after,blockquote:before,q:after,q:before{content:"";content:none}table{border-collapse:collapse;border-spacing:0}
```

Comment:
did you try to import it into your plugin? I think you should use mixin for it to work. remember that it must be localized to the plugin itself.

See this example:
1. add mixin method as shown below in the contrib-ui
2. @import that file and then `@include` the mixin under the root class name

```
@mixin side-panel {
  position: absolute;
  z-index: 1;
  overflow: hidden;
  transition: left $default-transition-time, opacity $default-transition-time, right $default-transition-time, bottom $default-transition-time,  top $default-transition-time;
  opacity: 0;
}

.vertical-side-panel {
  @include side-panel;
  height: 100%;
}

```